pyintegration/deephaven2/plot/figure.py

```diff
@@ -552,22 +552,22 @@ def chart_legend(
         j_figure = self.j_figure
 
         if {"color"}.issubset(non_null_args):
-            j_figure = self.j_figure.legendColor(color)
+            j_figure = j_figure.legendColor(color)
             non_null_args = non_null_args.difference({"color"})
             f_called = True
 
         if {"font"}.issubset(non_null_args):
-            j_figure = self.j_figure.legendFont(font)
+            j_figure = j_figure.legendFont(font)
             non_null_args = non_null_args.difference({"font"})
             f_called = True
 
         if {"font_family", "font_style", "font_size"}.issubset(non_null_args):
```

Comment:
Ok, I see here that my previous question about syntax wouldn't work as well here.  I'm still not sure how I feel about this syntax, though.  There are more "pythonic" ways to check for subsets in sets.  In particular, for font, font style, and font size, there may be a better way to handle it in a single if block rather than multiple.

Reply:
I'm closing this comment.  This can be revisited later on.